#! /bin/sh /usr/share/dpatch/dpatch-run
## 50_print-imagemagick-extensions.dpatch by Rafael Laboissiere <rafael@debian.org>
##
## DP: Call to cellidx will not fail when unknown file extensions imply
## DP: call to the convert command

@DPATCH@

--- octave2.9-2.9.9.orig/scripts/plot/print.m
+++ octave2.9-2.9.9/scripts/plot/print.m
@@ -203,7 +203,7 @@
   dev_list = {"aifm" "corel" "fig" "png" "pbm" "dxf" "mf" "hpgl", ...
 	      "ps" "ps2" "psc" "psc2" "eps" "eps2" "epsc" "epsc2" "emf"};
   convertname = "";
-  idx = cellidx (dev_list, dev);
+  [idx, errmsg] = cellidx (dev_list, dev);
   if (! idx)
     if (! isempty (devopt))
       convertname = strcat (devopt, ":", name);
